frontend/components/navbar/SearchBar.tsx

```diff
@@ -34,13 +36,13 @@ export default function SearchBar() {
   }, [pathname, searchParams]);
 
   const handleSearchClick = () => {
-    console.log("search");
+    searchBarLogger.info(`Event:${event}, Location:${location}`);
```

Comment:
I'm thinking we shouldn't need to log this unless we want metrics for future marketing on what are the most common search terms